scannerlib/proto/scan_instructions.proto

```diff
@@ -198,6 +198,7 @@ message GroupCriterion {
 message FileSet {
   // A single file.
   message SingleFile {
+    // Path to file. This value can be replaced with the previous check result if the PipelineToken is used in the Scan config
```

Comment:
I don't think this has anything to do with the ScanConfig? Would the following be more accurate? "If the wildcard %%pipeline%% is used, the value is replaced with the previous check result"

Reply:
done!